<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeographyWindow</class>
 <widget class="QDialog" name="GeographyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="lblbackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bGrounds/backgrounds/bber.jpg);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_QuestionDisplay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>921</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">;background-color: rgba(0,0,0, 120);
color: rgb(211, 211, 211);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_A">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover#pushButton_A{
	;;background-color: rgba(0,0,0, 120);
  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;

	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_A{
 border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	;background-color: rgba(0,0,0, 200);

 }

   
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_B">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover#pushButton_B{
	;;background-color: rgba(0,0,0, 120);
  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;

	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_B{
 border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	;background-color: rgba(0,0,0, 200);

 }

   
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_C">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover#pushButton_C{
	;;background-color: rgba(0,0,0, 120);
  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;

	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_C{
 border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	;background-color: rgba(0,0,0, 200);

 }

   
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_D">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>480</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover#pushButton_D{
	;;background-color: rgba(0,0,0, 120);
  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;

	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_D{
 border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	;background-color: rgba(0,0,0, 200);

 }

   
</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLive3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat:no-repeat;
background-image: url(:/bGrounds/backgrounds/h1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblScore">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SCORE: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLive1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat:no-repeat;
background-image: url(:/bGrounds/backgrounds/h1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLive2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat:no-repeat;

background-image: url(:/bGrounds/backgrounds/h1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_DisplayForTimer">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>20</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 45px;border-style: outset;padding: 10px;background-color: rgba(255, 255, 42,150);color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 20</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>481</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 22, 22);</string>
   </property>
   <widget class="QLabel" name="trophy_gif">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/bGrounds/backgrounds/trophy.gif);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Leave">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton:hover#pushButton_Leave{
;background-color: rgba(0,0,0, 200);

  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;
	font: 22pt &quot;Serif&quot;;
	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_Leave{
 border: 2px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	color: rgb(255, 255, 255);
	font: 15pt &quot;Serif&quot;;
	;;background-color: rgba(0,0,0, 100);

 }

   
</string>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MainMenu">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton:hover#pushButton_MainMenu{
;background-color: rgba(0,0,0, 200);

  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;
	font: 18pt &quot;Serif&quot;;
	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_MainMenu{
 border: 2px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	color: rgb(255, 255, 255);
	font: 15pt &quot;Serif&quot;;
	;;background-color: rgba(0,0,0, 100);

 }

   
</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Retry">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
 QPushButton:hover#pushButton_Retry{
;background-color: rgba(0,0,0, 200);

  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;
	font: 22pt &quot;Serif&quot;;
	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_Retry{
 border: 2px solid #555;
    border-radius: 10px;
    border-style: outset;
    padding: 5px;
	color: rgb(255, 255, 255);
	font: 15pt &quot;Serif&quot;;
	;;background-color: rgba(0,0,0, 100);

 }

   
</string>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CorrectAnswer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_WrongAnswers">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Score">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>pushButton_Leave</zorder>
   <zorder>pushButton_MainMenu</zorder>
   <zorder>pushButton_Retry</zorder>
   <zorder>trophy_gif</zorder>
   <zorder>label_CorrectAnswer</zorder>
   <zorder>label_WrongAnswers</zorder>
   <zorder>label_Score</zorder>
  </widget>
  <widget class="QLabel" name="anotherLabelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(13, 13, 13,160);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/icons/pause_w .png);
background-color: rgba(255, 255, 255,0);
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelpausebackground">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>270</width>
     <height>791</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,70);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Play">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>330</y>
     <width>130</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">

 QPushButton:hover#pushButton_Play{
	;;background-color: rgba(0,0,0, 120);
  border: 2px solid #555;
    border-radius: 15px;
    border-style: inset;
background-image: url(:/icons/icons/play_b.png);
background-repeat: no-repeat;
	color: rgb(255, 255, 255);
 }
QPushButton:!hover#pushButton_Play{

    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	
background-image: url(:/icons/icons/play_b.png);

background-color: rgba(255, 255, 255,0);
background-repeat: no-repeat;
 }

   
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="pauseText">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>229</y>
     <width>270</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 22, 22);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string> Game is Paused</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PauseRemaining">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>470</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 22, 22);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>  Pause Remaining: 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lblbackground</zorder>
  <zorder>widget</zorder>
  <zorder>lblLive1</zorder>
  <zorder>lblLive2</zorder>
  <zorder>lblLive3</zorder>
  <zorder>lblScore</zorder>
  <zorder>plainTextEdit_QuestionDisplay</zorder>
  <zorder>pushButton_A</zorder>
  <zorder>pushButton_B</zorder>
  <zorder>pushButton_C</zorder>
  <zorder>pushButton_D</zorder>
  <zorder>label_DisplayForTimer</zorder>
  <zorder>anotherLabelBackground</zorder>
  <zorder>labelpausebackground</zorder>
  <zorder>pauseButton</zorder>
  <zorder>pushButton_Play</zorder>
  <zorder>pauseText</zorder>
  <zorder>label_PauseRemaining</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
